recipe/migrations/cuda110.yaml

```diff
@@ -24,26 +24,27 @@ cxx_compiler_version:   # [linux]
   - 7                   # [linux64 or aarch64]
   - 8                   # [ppc64le]
 
-# cdt_name:  # [os.environ.get("BUILD_PLATFORM", "").startswith("linux") or (os.environ.get("CONFIG_VERSION", "1") == "1" and linux)]
-#   - cos6   # [os.environ.get("BUILD_PLATFORM") == "linux-64" or (os.environ.get("CONFIG_VERSION", "1") == "1" and linux64)]
-#   - cos7   # [os.environ.get("BUILD_PLATFORM") == "linux-aarch64" or (os.environ.get("CONFIG_VERSION", "1") == "1" and aarch64)]
-#   - cos7   # [os.environ.get("BUILD_PLATFORM") == "linux-ppc64le" or (os.environ.get("CONFIG_VERSION", "1") == "1" and ppc64le)]
-#   - cos7   # [os.environ.get("BUILD_PLATFORM") == "linux-armv7l" or (os.environ.get("CONFIG_VERSION", "1") == "1" and armv7l)]
-#
-#   - cos6   # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-#   - cos6   # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-#   - cos6   # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-#   - cos6   # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
-#   - cos7   # [linux64 and (os.environ.get("BUILD_PLATFORM") == "linux-64" or os.environ.get("CONFIG_VERSION", "1") == "1")]
+cudnn:
+  - undefined
+  - 7.6                 # [linux64]
+  - 7.6                 # [linux64]
+  - 7.6                 # [linux64]
+  - 7.6                 # [linux64]
```

Comment:
Should these be 7.6.5 similar to the `conda_build_config.yaml` pinnings?

Reply:
No, it should be `7` as that's the exports that we use.